model/registry/db.yaml

```diff
@@ -79,19 +79,12 @@ groups:
           The number of times a query was speculatively executed. Not set or `0` if the query was not executed speculatively.
         examples: [0, 2]
         tag: tech-specific-cassandra
-      - id: cassandra.table
+      - id: collection.name
         type: string
         stability: experimental
-        brief: The name of the primary Cassandra table that the operation is acting upon, including the keyspace name (if applicable).
-        note: >
-          This mirrors the db.sql.table attribute but references cassandra rather than sql.
-          It is not recommended to attempt any client-side parsing of
-          `db.statement` just to get this property, but it should be set if
-          it is provided by the library being instrumented.
-          If the operation is acting upon an anonymous table, or more than one table, this
-          value MUST NOT be set.
-        examples: 'mytable'
-        tag: tech-specific-cassandra
+        brief: The name of a collection (table, container) within the database.
```

Comment:
do we want to say something about it being either qualified or unqualified? or should we revisit once we add `db.collection.namespace` and then try to say something more about the relationship between `db.collection.namespace` and `db.collection.name`?

Reply:
I called it out in the note on the common `db.collection.name` with 

> If the collection name is parsed from the query, it SHOULD match the value provided in the query and may be qualified with the database name.

and also cleaned up yaml so it should now be reused for cassandra and other DBs where collection name could be parsed from the query

Reply:
moved the qualified sentence from requirement level to the note so it appears in the registry